event-sourcing/event-store-cassandra/src/test/scala/org/apache/james/eventsourcing/eventstore/cassandra/dto/OtherEvent.scala

```diff
@@ -7,21 +7,22 @@
  * "License"); you may not use this file except in compliance   *
  * with the License.  You may obtain a copy of the License at   *
  *                                                              *
- *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ * http://www.apache.org/licenses/LICENSE-2.0                   *
  *                                                              *
  * Unless required by applicable law or agreed to in writing,   *
  * software distributed under the License is distributed on an  *
  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
  * KIND, either express or implied.  See the License for the    *
  * specific language governing permissions and limitations      *
  * under the License.                                           *
- ****************************************************************/
+ * ***************************************************************/
+package org.apache.james.eventsourcing.eventstore.cassandra.dto
 
-package org.apache.james.eventsourcing.eventstore.cassandra;
+import org.apache.james.eventsourcing.{Event, EventId, TestAggregateId}
 
-public interface CassandraEventStoreTable {
-    String EVENTS_TABLE = "eventStore";
-    String AGGREGATE_ID = "aggregateId";
-    String EVENT = "event";
-    String EVENT_ID = "eventId";
-}
+final case class OtherEvent(override val eventId: EventId, aggregateId: TestAggregateId, payload: Long) extends Event {
```

Comment:
Because it does not make sense to extend case class. In fact it can cause issue with equals and hashcode methode when comparing subclasses. It is recommended to make them final